api/k8s/v1/k8s.proto

```diff
@@ -801,6 +809,27 @@ message DeleteJobRequest {
 message DeleteJobResponse {
 }
 
+message JobConfig {
```

Comment:
is this wrapper necessary? seems like we could just use `google.Protobuf.Value job_config` in the request.

Reply:
Yep we could, I just thought the naming will help with understanding what the value mean, but I could change it and add  a comment if its less confusing

Reply:
Followed the pattern from here for consistency: https://github.com/lyft/clutch/blob/main/api/envoytriage/v1/output.proto#L24. Let me know if you prefer I change it

Reply:
that was done because we might extract variables from the dump later out into the message in addition to the raw content. but in this case we already have a higher level object.

i agree though it works either way so you can go with your preference.

my other question i guess is `Config` the right name for this? would `Spec` be a more k8s-ish name?

Reply:
The Job object has a field within it called Spec, so I didn't use spec cause I thought it would confuse the user that they only have to provide the spec blob